tests/integration/003_configuration_variable/main.tf

```diff
@@ -58,3 +58,19 @@ output "tested3_enum_2" {
   value     = data.env0_configuration_variable.tested3.enum[1]
   sensitive = true
 }
+
+resource "env0_configuration_variable" "regex_var" {
+  project_id = env0_project.test_project.id
+  name       = "regex_var"
+  regex      = "^test-\\d+$"
+}
+
+data "env0_configuration_variable" "regex_var" {
+  project_id = env0_project.test_project.id
+  name       = "regex_var"
+  depends_on = [env0_configuration_variable.regex_var]
+}
+
+output "regex" {
+  value = data.env0_configuration_variable.regex_var.regex
+}
```

Comment:
Add enter

Reply:
[new line at the end](https://github.com/env0/terraform-provider-env0/pull/370/commits/51e37e675c02ca154dd416ac094e3449aa88abc7)